### ISSUE TITLE
[Accessibility visual improvement] Color contrast failure for Tabs / Counter Component [only in dark mode] (MAS ID: 260779) 

### ISSUE THREAD

**Issue by ckbedwell** [grafana/grafana#72868]
**Steps to reproduce**:
Go to Tabs story in Storybook documentation: https://developers.grafana.com/ui/latest/index.html?path=/story/layout-tabs--counter. Either run an accessibility checker or use dev tools to see the color contrast is below the recommended threshold.

**Actual Result**:
Result is 4.16 on the color contrast in dark mode.

**Expected Result**
It should be at least 4.5:1. 
![image](https://github.com/grafana/grafana/assets/6906380/bf0d5b9e-45ed-4adf-922f-53d206adae51)

**Comment by GunFood** [comment]
Still needed
